README.md

```diff
@@ -77,15 +87,17 @@ poetry run pytest tests
 
 ## Trying It Out
 
-We encourage interested parties to try it out and give us feedback. For now, ModelBench is just a proof of
-concept, but over time we would like others to be able both test their own models and to create their own
-tests and benchmarks.
+We encourage interested parties to try it out and give us feedback. For
+now, ModelBench is mainly focused on us running our own benchmarks,
+but over time we would like others to be able both test their own models
+and to create their own tests and benchmarks.
```

Comment:
Is this true still?

Reply:
I think it's less true, but not yet false, and since they're still excited about partnering with other organizations, let's leave it in for now.